README.md

```diff
@@ -31,15 +31,15 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [Portfolio] <a name="about-project"></a>
+# 📖 [Capstone Project] <a name="about-project"></a>
 
-**[Portfolio]** is a unique way to showcase your work and let others know about yourself.
+**[Capstone Project]** is a unique way to showcase your work and let others know about yourself.
```

Comment:
- [optional] kindly add more information on what your project is all about. This helps a user to have a clear idea about it just from the README file

Reply:
Fixed.